src/app/store/index.ts

```diff
@@ -0,0 +1,77 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { atomWithStore } from 'jotai/redux';
+import devToolsEnhancer from 'remote-redux-devtools';
+import { AnyAction, combineReducers, configureStore, ThunkAction } from '@reduxjs/toolkit';
+import {
+  persistStore,
+  persistReducer,
+  FLUSH,
+  REHYDRATE,
+  PAUSE,
+  PERSIST,
+  PURGE,
+  REGISTER,
+} from 'redux-persist';
+
+import { keySlice } from './keys/key.slice';
+import { stxChainSlice } from './chains/stx-chain.slice';
+import { broadcastActionTypeToOtherFramesMiddleware } from './utils/broadcast-action-types';
+import { inMemoryKeySlice } from './in-memory-key/in-memory-key.slice';
+import { ExtensionStorage } from './utils/extension-storage';
+
+const storage = new ExtensionStorage(chrome.storage.local, chrome.runtime);
+
+const rootReducer = combineReducers({
+  keys: keySlice.reducer,
+  chains: combineReducers({
+    stx: stxChainSlice.reducer,
+  }),
+  inMemoryKeys: inMemoryKeySlice.reducer,
+});
+
+const persistConfig = {
+  key: 'root',
+  version: 1,
+  storage,
+  serialize: true,
+  whitelist: ['keys', 'chains'],
+};
+
+const persistedReducer = persistReducer(persistConfig, rootReducer);
+
+export const store = configureStore({
+  reducer: persistedReducer,
+  middleware: getDefaultMiddleware => [
+    ...getDefaultMiddleware({
+      serializableCheck: {
+        ignoredActions: [FLUSH, REHYDRATE, PAUSE, PERSIST, PURGE, REGISTER],
```

Comment:
Why do we ignore these?

Reply:
https://redux-toolkit.js.org/usage/usage-guide#use-with-redux-persist